libs/fleet-mcp/src/fleet_mcp/__main__.py

```diff
@@ -341,6 +341,44 @@ async def show_agent_log(
     return await show_logs_impl(get_task_service(), agent_name, page, page_size)
 
 
+# ========================================================================
+# Resources
+# ========================================================================
+
+
+@mcp.resource("config://coder_metadata")
+def get_coder_metadata_config() -> str:
```

Comment:
Implement the metadata inside the coder workspace template using Terraform https://registry.terraform.io/providers/coder/coder/latest/docs/resources/metadata, not by adding a MCP tool call.